keras_core/utils/layer_utils.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
+from keras_core.utils.module_utils import tensorflow as tf
+
+
+def validate_string_arg(
```

Comment:
I've recently added this function to `keras_core/utils/argument_validation.py`, so you can remove this file

Reply:
@fchollet sure removing it.